test/instances/aerodrome/AerodromeLpHyperdriveInstanceTest.t.sol

```diff
@@ -0,0 +1,174 @@
+        uint256 baseAmount
+    ) internal pure override returns (uint256) {
+        return AerodromeLpConversions.convertToShares(baseAmount);
+    }
+
+    /// @dev Converts share amount to the equivalent amount in base.
+    /// @param shareAmount The share amount.
+    /// @return The converted base amount.
+    function convertToBase(
+        uint256 shareAmount
+    ) internal pure override returns (uint256) {
+        return AerodromeLpConversions.convertToBase(shareAmount);
+    }
+
+    /// @dev Deploys the AerodromeLp Hyperdrive deployer coordinator contract.
+    /// @param _factory The address of the Hyperdrive factory.
+    /// @return The coordinator address.
+    function deployCoordinator(
+        address _factory
+    ) internal override returns (address) {
+        vm.startPrank(alice);
+        return
+            address(
+                new AerodromeLpHyperdriveDeployerCoordinator(
+                    string.concat(config.name, "DeployerCoordinator"),
+                    _factory,
+                    address(new AerodromeLpHyperdriveCoreDeployer(gauge)),
+                    address(new AerodromeLpTarget0Deployer(gauge)),
+                    address(new AerodromeLpTarget1Deployer(gauge)),
+                    address(new AerodromeLpTarget2Deployer(gauge)),
+                    address(new AerodromeLpTarget3Deployer(gauge)),
+                    address(new AerodromeLpTarget4Deployer(gauge))
+                )
+            );
+    }
+
+    /// @dev Fetches the total supply of the base and share tokens.
+    /// @return The total supply of base.
+    /// @return The total supply of vault shares.
+    function getSupply() internal view override returns (uint256, uint256) {
+        return (
+            config.baseToken.balanceOf(address(gauge)),
+            config.baseToken.balanceOf(address(gauge))
+            // gauge.totalSupply()
+            // config.baseToken.balanceOf(address(hyperdrive))
+        );
+    }
+
+    /// @dev Fetches the token balance information of an account.
+    /// @param account The account to query.
+    /// @return The balance of base.
+    /// @return The balance of vault shares.
+    function getTokenBalances(
+        address account
+    ) internal view override returns (uint256, uint256) {
+        return (
+            config.baseToken.balanceOf(account),
+            gauge.balanceOf(address(account))
+        );
+    }
+
+    /// Getters ///
+
+    /// @dev Test the instances getters.
+    function test_getters() external view {
+        (, uint256 totalShares) = getTokenBalances(address(hyperdrive));
+        assertEq(hyperdrive.totalShares(), totalShares);
+    }
+
+    /// Price Per Share ///
+
+    /// @dev Fuzz test that verifies that the vault share price is the price
+    ///      that dictates the conversion between base and shares.
+    /// @param basePaid The fuzz parameter for the base paid.
+    function test__pricePerVaultShare(uint256 basePaid) external {
```

Comment:
We should have a test for the rewards. If you look at `test/instances/stk-well`, you'll see what I'm talking about.